wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -987,6 +1022,14 @@ impl PhysicalDeviceProperties {
                 .min(crate::MAX_VERTEX_BUFFERS as u32),
             max_vertex_attributes: limits.max_vertex_input_attributes,
             max_vertex_buffer_array_stride: limits.max_vertex_input_binding_stride,
+            min_subgroup_size: self
+                .subgroup_size
+                .map(|subgroup_size| subgroup_size.min_subgroup_size)
+                .unwrap_or(0),
```

Comment:
I think these can be 4 and 128 if size control is not available (Vulkan 1.1 and friends).